src/components/header/header.module.css

```diff
@@ -0,0 +1,123 @@
+*{
```

Comment:
put this in index.css, as its a main style

Reply:
done